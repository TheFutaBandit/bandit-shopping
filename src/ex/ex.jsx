export const Ex = () => {
    return <h1>Hello World</h1>;
}
