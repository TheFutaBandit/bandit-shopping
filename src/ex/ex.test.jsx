import {describe, expect, it} from 'vitest'
import {render, screen} from '@testing-library/react'
import {Ex} from './ex'

describe("initial test", () => {
    it("first test", () => {
        const { container } = render(<Ex />)
        expect(container).toMatchSnapshot();
    })
})