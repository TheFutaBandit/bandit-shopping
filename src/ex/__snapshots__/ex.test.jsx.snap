// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`initial test > first test 1`] = `
<div>
  <h1>
    Hello World
  </h1>
</div>
`;
